travis/test.sh

```diff
@@ -10,7 +10,7 @@ flutter analyze --flutter-repo --no-current-directory --no-current-package --con
 flutter test --flutter-repo
 
 (cd packages/cassowary; pub run test -j1)
-# (cd packages/flutter_sprites; ) # No tests to run.
+(cd packages/flutter_sprites; flutter test)
```

Comment:
I take this back. I tried to add this to flutter test but flutter test really wants to run in just one package at a time. We could make it more generic but there's really no reason to.